lm-api/lm_api/permissions.py

```diff
@@ -8,12 +8,16 @@ class Permissions(str, Enum):
 
     ADMIN = "license-manager:admin"
 
+    STATUS_UPDATE = "license-manager:status:update"
+    STATUS_READ = "license-manager:status:read"
+
     LICENSE_SERVER_CREATE = "license-manager:license-server:create"
     LICENSE_SERVER_READ = "license-manager:license-server:read"
     LICENSE_SERVER_UPDATE = "license-manager:license-server:update"
     LICENSE_SERVER_DELETE = "license-manager:license-server:delete"
 
     CONFIG_CREATE = "license-manager:config:create"
+    CONFIG_VIEW = "license-manager:config:view"
```

Comment:
Why is this view permission being added back in?

Reply:
I added it for testing, since I don't have a token with the new permissions.
Forgot to remove, thank you for catching it.